tests/sanity/extra/licenses.py

```diff
@@ -30,7 +30,8 @@ def find_licenses(filename, relax=False):
                 print('%s: found copyright line with "Copyright:". Please remove the colon.' % (filename, ))
             idx = line.find('SPDX-License-Identifier: ')
             if idx >= 0:
-                spdx_license_identifiers.append(line[idx + len('SPDX-License-Identifier: '):])
+                lic_id = line[idx + len('SPDX-License-Identifier: '):]
+                spdx_license_identifiers.extend(lic_id.split(' OR '))
```

Comment:
This should at least account for identifiers split with `AND` if not also license identifiers that have groupings with multiple boolean conditions. It would obviously require more code, but future proofing for if collections do start using this is a good idea in my book.

Reply:
If you implement grouping, you'll also want to remove the `found identical SPDX-License-Identifier values` check.

Reply:
I think I only want to implement the bare minimum to support what's needed in this collection. The default for files should be to have exactly one license, with any other combination (like `OR`) reserved for very special circumstances.